lib/fault_handler/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig FAULT_HANDLER
+	bool "Fault handler implementation for Nordic DKs"
+	default y
+	help
+	  Enable using the fault handler definied for Nordic DKs
```

Comment:
Instead of a generic fault handler let's have this handler specifically trigger resets.

So

`menuconfig RESET_ON_FAULT`

or similar.

Reply:
Ok. I agree on that